### PR TITLE
Convert Toolbar to Flux

### PR DESCRIPTION
**Pull request by krazemon**
Continuing the Flux domination one component at a time.

### CODE REVIEW

File: js/components/Toolbar.jsx [hunk 1]
```diff
@@ -87,6 +64,7 @@ var Toolbar = React.createClass({
           <Link to="profile" params={{userId: this.state.currentUser._id}}>{this.state.currentUser.firstname}</Link>
```
Comment (ritmatter) [inline js/components/Toolbar.jsx]: So on the render, when we get state from stores, why don't we have to check if we have this.state.currentUser, this.state.user, etc? Won't render be called before the ajax calls come back, in which case all of the state variables will be undefined?

Comment (ritmatter) [inline js/components/Toolbar.jsx]: Ahh, the code editing hid it but it looks like we actually do still check for the currentUser.

File: js/constants/RepcoinConstants.js [hunk 2]
```diff
@@ -3,7 +3,16 @@ var keyMirror = require('keymirror');
 module.exports = {
   ActionTypes: keyMirror({
     GET_CATEGORIES: null,
-    RECEIVE_CATEGORIES: null
+    RECEIVE_CATEGORIES: null,
+    GET_CURRENT_USER: null,
+    RECEIVE_CURRENT_USER: null,
+    GET_LOGGED_IN: null,
+    RECEIVE_LOGGED_IN: null,
```
Comment (ritmatter) [inline js/constants/RepcoinConstants.js]: Why do all these hash to null?

**Comment by ritmatter** [comment]
Just answer my questions when you get a chance, I'm pretty confused about where we're housing all of these constants

LGTM upon clarification

File: js/constants/RepcoinConstants.js (hunk 2 above)
Comment (krazemon) [inline js/constants/RepcoinConstants.js]: keymirror returns an object with the nulls equal to the string versions of those variable names.

Comment (krazemon) [inline js/constants/RepcoinConstants.js]: https://github.com/STRML/keyMirror

**Pull request merged**